setup.py

```diff
@@ -115,6 +115,12 @@ def _main():
     with open("README.md") as f:
         long_description = f.read()
 
+    # ROCm build of torchaudio needs libomp.so to be packaged with the wheel
+    # to avoid import-time error due to missing dependency
+    if torch.hip is not None:
```

Comment:
this will affect all builds not only rocm. Since this issue happens only on rocm can we implement this only for rocm builds ?

Reply:
Actually this should be `torch.version.hip is not None`. That should ensure that this change is applied only to ROCm builds. Doest that sound good?
@pruthvistony, can you please make the changes as appropriate?

Reply:
changes pushed.